bessctl/cli.py

```diff
@@ -441,6 +430,52 @@ def restore_echoctl(self):
         except:
             pass
 
+    def go_interactive(self):
```

Comment:
Yeah, I agree that the name could be a bit misleading.  The actual semantics -- that it doesn't try to go into interactive if it's already in interactive -- was certainly the more important aspect!